src/_data/partners.json

```diff
@@ -0,0 +1,62 @@
+[
+  {
+      "name": "BangPypers",
+      "link": "https://bangalore.pythonindia.org/",
+      "image": "/img/partners/bangpypers.png"
+    },
+    {
+      "name": "HydPy",
+      "link": "https://www.hydpy.org/",
+      "image": "/img/partners/hydpy.png"
+    },
+    {
+      "name": "PythonPune",
+      "link": "https://pythonpune.in/",
+      "image": "/img/partners/PythonPune_banner.svg"
```

Comment:
Can you use this logo with gray text: https://github.com/pythonpune/design/blob/master/logo/full-logo/PythonPune_logo_90dpi.png

Reply:
@bhavin192 done